app/queries/get_references_to_chase.rb

```diff
@@ -1,9 +1,17 @@
 class GetReferencesToChase
   def self.call
-    ApplicationReference
-      .feedback_requested
-      .where(['requested_at < ?', chase_referee_time_limit])
-      .where.not(id: ChaserSent.reference_request.select(:chased_id))
+    if FeatureFlag.active?(:decoupled_references)
+      ApplicationReference
+        .feedback_requested
+        .where(['requested_at < ?', chase_referee_time_limit])
+        .where(application_form_id: ApplicationForm.where(submitted_at: nil).pluck(:id))
+        .where.not(id: ChaserSent.reference_request.select(:chased_id))
```

Comment:
Does this seem correct? Getting all references past the time limit, whose application forms have not been submitted and have not previously been chased.

Reply:
This looks correct to me, although the explicit check on `submitted_at` hints that we probably also need to cancel any outstanding reference requests when a candidate submits their application. With this logic we won't send chasers for those but it does seem like a loose end to tie up. Will check and write a card if needed.

Reply:
Yea this is now being picked up via a separate card.